### PR TITLE
Add tolerance parameter to WebHooks.contructEvent

### PR DESCRIPTION
**Pull request by kamagatos**
If changing an existing definition:

 Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/stripe/stripe-node/blob/2e0090559e96f16e9a6f057dadde135f0c731488/lib/Webhooks.js#L9

### CODE REVIEW

File: types/stripe/index.d.ts [hunk 1]
```diff
@@ -7192,7 +7192,7 @@ declare namespace Stripe {
         }
 
         class WebHooks {
-            constructEvent<T>(requestBody: any, signature: string | string[], endpointSecret: string): webhooks.StripeWebhookEvent<T>;
+            constructEvent<T>(requestBody: any, signature: string | string[], endpointSecret: string, tolerance: number): webhooks.StripeWebhookEvent<T>;
```
Comment (LinusU) [inline types/stripe/index.d.ts]: The `tolerance` argument should be optional

**Review by kamagatos — COMMENTED** [review]
Made the parameter optional

File: types/stripe/index.d.ts (hunk 1 above)
Comment (kamagatos) [reply types/stripe/index.d.ts]: Thanks!

**Review by wjohnsto — APPROVED** [review]
Looks good 👍

**Review by LinusU — APPROVED** [review]

**Pull request merged**